packages/shared.test/src/handlers/actionEndpoints.ts

```diff
@@ -121,7 +106,8 @@ async function handleAction<
     actionResponseMap[actionType][stableStringify(parsedBody)];
 
   if (
-    req.params.ontologyApiName === defaultOntology.apiName
+    (req.params.ontologyApiName === defaultOntology.apiName
+      || req.params.ontologyApiName === defaultOntology.rid)
```

Comment:
Correct. The gateway allows this and is part of how we can switch everything over cleanly. But now that I've updated the code to pass the rid instead of the ontologyApi name, all the mocks broke.